bindings/matrix-sdk-ffi/src/sliding_sync.rs

```diff
@@ -139,6 +156,26 @@ impl SlidingSyncRoom {
         &self,
         listener: Box<dyn TimelineListener>,
     ) -> Option<Arc<StoppableSpawn>> {
+        Some(Arc::new(self.add_timeline_listener_inner(listener)?))
+    }
+
+    pub fn subscribe_and_add_timeline_listener(
```

Comment:
It feels a bit weird that this lives just in the bindings, wouldn't this be useful to Fractal as well?